modules/pipeline/pipengine/reconciler/taskrun/taskop/start.go

```diff
@@ -159,6 +164,19 @@ func (s *start) TuneTriggers() taskrun.TaskOpTuneTriggers {
 	}
 }
 
+// injectJobID save flink, spark job id after start
+func injectJobID(tr *taskrun.TaskRun, data interface{}) error {
```

Comment:
injectJobIDForFlinkSpark is better.

Reply:
only dcos return str job id,already remove injectJobID